src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -881,7 +891,8 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
     integer(i_kind),dimension(:,:),allocatable:: uvlnames
     integer(i_kind):: inner_vars,numfields
     integer(i_kind):: ndynvario2d,ntracerio2d,ilev,jdynvar,jtracer
-    integer(r_kind):: iuv,ndynvario3d,ntracerio3d,ntracerchemio3d
+    integer(r_kind):: iuv,ndynvario3d,ntracerio3d
```

Comment:
Why is this integer declared to be type `r_kind`?

Reply:
great catch! and that was already in develop...

Reply:
I thought this has been resolved since in a previous version of my code for CMAQ it was fixed. Good to fix though It doesn't affect results.